client/src/components/DataFiles/DataFilesModals/DataFilesPreviewModal.js

```diff
@@ -56,7 +62,7 @@ const DataFilesPreviewModal = () => {
       >
         <ModalHeader toggle={toggle}>File Preview: {params.name}</ModalHeader>
         <ModalBody>
-          {isLoading && <PreviewModalSpinner />}
+          {(isLoading || isFrameLoading) && <PreviewModalSpinner />}
```

Comment:
this needs to consider the cases when we aren't using the iframe and therefore don't need to worry about `isFrameLoading`).  here is a suggested change:

```suggestion
          {(isLoading || (isFrameLoading && !previewContent)) && <PreviewModalSpinner />}
```

(🎗️  i'll make sure to check this when this gets merged and I merge master to https://github.com/TACC/Frontera-Portal/pull/300 ).